frontend/src/metabase/containers/UndoListing.jsx

```diff
@@ -79,8 +106,20 @@ function UndoToast({ undo, onUndo, onDismiss }) {
           data-testid="toast-undo"
           color={undo.toastColor}
           role="status"
+          noBorder={undo.showProgress}
           style={styles}
+          className={CS.toast}
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
         >
+          {undo.showProgress && (
+            <Progress
+              size="sm"
+              color={paused ? "bg-dark" : "brand"}
+              value={100}
```

Comment:
This breaks a11y - `value` should be the actual value.
We should also have `aria-label` here.

BTW the CSS animation is only roughly in sync with JS - I experienced at least 1 "jump" in the animation while testing this.

Reply:
I agree, but as far as I understand, we don't need to have the exact value to be in sync with animation, but to provide more or less clear picture that user needs to hurry up. 

My initial approach was in utilizing Transform component from mantine, but in this case I didn't see a way to stop the animation without keeping interval in the state. Seems same solution will be required to fix a11y

Reply:
@kamilmielnik I reworked animation to js in [`089c2aa` (#44015)](https://github.com/metabase/metabase/pull/44015/commits/089c2aae72c09e0fa4d924ca31ce12150b0bbd17) please take a look one more time. I still can see toast to disappear a bit earlier than a bar, but now no jumps